docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -6,210 +6,229 @@ description: Secretless Broker Documentation
-To create the certificate, you can run the following commands (as described in
-the [PostgreSQL documentation](https://www.postgresql.org/docs/9.6/ssl-tcp.html)):
-```bash
-openssl req -new -x509 -days 365 -nodes -text -out server.crt \
-  -keyout server.key -subj "/CN=pg"
-chmod og-rwx server.key
-```
-<pre>Generating a 2048 bit RSA private key
-....................................................................................+++++
-.......+++++
-writing new private key to 'server.key'
------</pre>
-
-Now that the certificates have been created, you will proceed to store them in Kubernetes secrets. This is better than hard-coding them - but in a real deployment you would want to store your secrets in a fully-featured vault, like Conjur.
-
-Run this code to store the certificate files in a Kubernetes secret in the
-`quick-start-backend-ns` namespace:
-```bash
-kubectl --namespace quick-start-backend-ns \
-  create secret generic \
-  quick-start-backend-certs \
-  --from-file=server.crt \
-  --from-file=server.key
-```
-<pre>secret "quick-start-backend-certs" created</pre>
-
-**3.** To create and save the **PostgreSQL StatefulSet** manifest in a file named **pg.yml** in your current working directory, run the command:
-
-```bash
-cat << EOF > pg.yml
-apiVersion: apps/v1
-kind: StatefulSet
-metadata:
-  name: pg
-  labels:
-    app: quick-start-backend
-spec:
-  serviceName: quick-start-backend
-  selector:
-    matchLabels:
-      app: quick-start-backend
-  template:
+To deploy a PostgreSQL StatefulSet:
+
+1. Create a dedicated **namespace** for the storage backend:
+    ```bash
+    kubectl create namespace quick-start-backend-ns
+    ```
+    <pre>
+    namespace "quick-start-backend-ns" created
+    </pre>
+
+1. Create a self-signed certificate (see [PostgreSQL documentation for
+   more](https://www.postgresql.org/docs/9.6/ssl-tcp.html)):
+
+    ```bash
+    openssl req -new -x509 -days 365 -nodes -text -out server.crt \
+      -keyout server.key -subj "/CN=pg"
+    chmod og-rwx server.key
+    ```
+    <pre>Generating a 2048 bit RSA private key
+    ....................................................................................+++++
+    .......+++++
+    writing new private key to 'server.key'
+    -----</pre>
+
+1. Store the certificate files as Kubernetes secrets in the
+   `quick-start-backend-ns` namespace:
+    ```bash
+    kubectl --namespace quick-start-backend-ns \
+      create secret generic \
+      quick-start-backend-certs \
+      --from-file=server.crt \
+      --from-file=server.key
+    ```
+    <pre>secret "quick-start-backend-certs" created</pre>
+
+    **Note:** _While Kubernetes Secrets are more secure than hard-coded ones, in
+    a real deployment you should secure secrets in a fully-featured vault, like
+    Conjur._
+
+1. Create and save the **PostgreSQL StatefulSet manifest** in a file named **pg.yml** in your current working directory:
+
+    ```bash
```

Comment:
copy/pasting this didn't work

Reply:
removing the extra initial space on each line and running worked, so this does appear to be the issue